Sample Applications/WPFGallery/MainWindow.xaml.cs

```diff
@@ -168,4 +168,17 @@ private void OnNavigating(object? sender, NavigatingEventArgs e)
             }
         }
     }
+
+    private void ControlsList_PreviewKeyDown(object sender, KeyEventArgs e)
+    {
+        if (e.Key == Key.Enter) 
```

Comment:
Shouldn't we mark the event handled in this scenario ?
Like 
`e.Handled = true`